dotnet/Capstone/Controllers/BirdNoteController.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Capstone.DAO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System;
+using Capstone.Models;
+using Capstone.Exceptions;
+using System.Security.Principal;
+
+namespace Capstone.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BirdNoteController : ControllerBase
```

Comment:
The following methods in BirdNoteController print console strings instead of returning error messages:
- getNotes
- getNote

Reply:
Done